src/PerformanceTests/Common/Scenarios/SagaUpdate/SagaUpdateRunner.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+partial class SagaUpdateRunner
+    : BaseRunner
+{
+    public static bool Shutdown;
+    readonly ILog Log = LogManager.GetLogger(nameof(SagaUpdateRunner));
+
+    protected override async Task Start(ISession session)
+    {
+        var maxConcurrencyLevel = ConcurrencyLevelConverter.Convert(Permutation.ConcurrencyLevel);
+        var seedSize = maxConcurrencyLevel * 2;
```

Comment:
I've seen this quite a number of times. Isn't this something you could do in BaseRunner or some helper class perhaps?